plugins/chat/assets/stylesheets/common/chat-unread-indicator.scss

```diff
@@ -11,6 +11,8 @@
   text-align: center;
   transition: border-color linear 0.15s;
 
+  @content;
```

Comment:
what does this refer to? (same in the chat-scrollbar file)

Reply:
it's SCSS/SASS equivalent of `yield`:

```scss
@mixin foo {
  color: red;
  padding-inline: 1rem;

  @contents;
}

#bar {
  margin-top: 1rem;

  @include foo {
    color: blue;
  }
}
```

I'm using it to highlight props that specifically override mixin defaults (and to avoid `mixed-decls` issue)